Hardcore.js

```diff
@@ -0,0 +1,205 @@
+    .sendKeys('4');
+
+  // set series
+  let selectSeries = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_71"]/span[1]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseTypeOfSeries = await driver
+    .findElement(By.xpath('//*[@id="select_option_216"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  // Instance type: n1-standard-8    (vCPUs: 8, RAM: 30 GB)
+  let selectMachineType = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_72"]/span[1]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseMachineType = await driver
+    .findElement(By.xpath('//*[@id="select_option_419"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  //Выбрать Add GPUs
+  // * Number of GPUs: 1
+  // * GPU type: NVIDIA Tesla V100
+  let selectAddGpus = await driver
+    .findElement(
+      By.xpath(
+        '//*[@id="mainForm"]/div[2]/div/md-card/md-card-content/div/div[1]/form/div[11]/div[1]/md-input-container/md-checkbox/div[2]'
+      )
+    )
+    .click();
+  await driver.sleep(2000);
+
+  let selectGpuType = await driver
+    .findElement(By.xpath('//*[@id="select_462"]'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseGpuType = await driver
+    .findElement(By.xpath('//*[@id="select_option_469"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  let selectNumberOfGpus = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_451"]/span[1]'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseNumberOfGpus = await driver
+    .findElement(By.xpath('//*[@id="select_option_473"]'))
+    .click();
+  await driver.sleep(2000);
+
+  // Local SSD: 2x375 Gb
+  let selectLocalSsd = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_413"]/span[1]'))
+    .click();
+  await driver.sleep(2000);
+
+  let choosetLocalSsd = await driver
+    .findElement(By.xpath('//*[@id="select_option_440"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  // Datacenter location: Frankfurt (europe-west3)
+  let selectDatacenterLocation = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_74"]/span[1]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseDatacenterLocation = await driver
+    .findElement(By.xpath('//*[@id="select_option_237"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  // Commited usage: 1 Year
+  let selectCommitedUsage = await driver
+    .findElement(By.xpath('//*[@id="select_value_label_75"]/span[1]'))
+    .click();
+  await driver.sleep(2000);
+
+  let chooseCommitedUsage = await driver
+    .findElement(By.xpath('//*[@id="select_option_114"]/div'))
+    .click();
+  await driver.sleep(2000);
+
+  // 7. Нажать Add to Estimate
+  let selectButtonAdd = await driver
+    .findElement(
+      By.xpath(
+        '//*[@id="mainForm"]/div[2]/div/md-card/md-card-content/div/div[1]/form/div[19]/button'
+      )
+    )
+    .click();
+  await driver.sleep(2000);
+
```

Comment:
Please make changes for locators before this line as it marked in Hurt-Me-Plenty.js